app/src/main/java/com/ucm/informatica/spread/Model/Region.java

```diff
@@ -1,4 +1,28 @@
 package com.ucm.informatica.spread.Model;
 
+
```

Comment:
Define a model for polygons as list of coordinates of the vertices and a counter to see how many markers lie on it